release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -0,0 +1,1810 @@
+[id="ocp-4-9-release-notes"]
```

Comment:
Do you need this file?

Reply:
Not sure why this ocp file is included in this PR. You are correct, it is not a part of this guide. I will investigate.

Reply:
@tmalove, this file is still present.

Reply:
I think this needs to be removed before merging iiuc

/remove-label merge-review-in-progress
/remove-label merge-review-needed

Reply:
Done.